src/internal/group_api/mod.rs

```diff
@@ -526,6 +530,112 @@ pub async fn group_create<CR: rand::CryptoRng + rand::RngCore>(
     resp.try_into()
 }
 
+#[derive(Debug, Clone)]
+pub struct GroupUpdatePrivateKeyResult {
+    needs_rotation: bool,
+}
+
+impl GroupUpdatePrivateKeyResult {
+    /// True if this group's private key requires additional rotation
+    pub fn needs_rotation(&self) -> bool {
+        self.needs_rotation
+    }
+}
+
+/// Maps the successful results of `transform::encrypt_to_with_key()` into a vector of GroupAdmins
+fn collect_request_admins(
```

Comment:
Not a huge deal, but the name of this function made me confused for quite a while. Assuming that I understand what it does, maybe `collect_group_admin_keys` or similar? I think I'm mostly just confused about what "request_admins" means.

Reply:
Sure, I can make that change. "request_admins" was just referring to the requests::GroupAdmin that we needed a vector of when sending the request.